src/tiled/commandbutton.cpp

```diff
@@ -34,6 +35,7 @@ using namespace Tiled::Internal;
 CommandButton::CommandButton(QWidget *parent)
     : QToolButton(parent)
     , mMenu(new QMenu(this))
+    , mCommandManager(new CommandManager(this))
```

Comment:
I could pass the singleton instance, but that would lead to passing it via `TilesetEditor` and `MapEditor` to `MainToolbar` to `CommandButton` which won't be an effective solution, I think.